docs/site/content/docs/latest/ref-unmanaged-cluster.md

```diff
@@ -0,0 +1,201 @@
+# Unmanaged Clusters Reference
+
+This is the reference documentation for the `unmanaged-cluster` plugin. If
+you're trying to use unmanaged clusters for the first time, see
+[Getting Started with Unmanaged Clusters](getting-started-unmanaged/).
+
+## Command aliases
+
+As a Tanzu CLI plugin, unmanaged clusters can be interacted with using `tanzu
+unmanaged-cluster`. `uc`, `um`, and `unmanaged` are all valid aliases for the
+command. This means the following commands equate to the same:
+
+* `tanzu unmanaged-cluster create hello`
+* `tanzu uc create hello`
+* `tanzu um create hello`
+* `tanzu unmanaged create hello`
+
+## Creating clusters
+
+`create` is used to create a new cluster. By default, it:
+
+1. Installs a cluster using `kind`.
+1. Installs `kapp-controller`.
+1. Installs a core package repository.
+1. Installs a user-managed package repository.
+1. Installs a CNI package.
+    * defaults to `antrea`.
+1. Sets your kubeconfig context to the newly created cluster.
+
+To create a cluster, run:
+
+```sh
+tanzu unmanaged-cluster create ${CLUSTER_NAME}
+```
+
+## Listing clusters
+
+`list` or `ls` is used to list all known clusters. To list known clusters, run:
+
+```sh
+tanzu unmanaged-cluster list
+```
+
+## Deleting clusters
+
+`delete` or `rm` is used to delete a cluster. It will:
+
+1. Attempt to delete the cluster based on the provider.
+    * by default, clusters use `kind`, this will delete the `kind` cluster.
+1. Attempt to remove the cluster's directory.
+    * located at `~/.config/tanzu/tkg/unmanaged/${CLUSTER_NAME}`.
```

Comment:
very small nit: since this is a _directory_ should this be:

```diff
+ ~/.config/tanzu/tkg/unmanaged/${CLUSTER_NAME}/
- ~/.config/tanzu/tkg/unmanaged/${CLUSTER_NAME}
```